ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,9 @@
 <!-- 
-   If you've just upgraded to 8.0, please read the upgrade guide first (https://docs.automapper.org/en/latest/8.0-Upgrade-Guide.html).
--->
-<!-- For feature requests, just clear out the below 
-Try to provide [a minimal, complete, and verifiable example](https://stackoverflow.com/help/mcve), preferably a [gist](https://gist.github.com/lbargaoanu/9c7233441c3a3413cc2b9b9ebb5964a9) that we can execute and see fail. [Here](https://gist.github.com/lbargaoanu/0cbc531306223f7ffc5468becf2642d6) is an example for ProjectTo.
+
+   If you're new to AutoMapper, please ask a question on StackOverflow first and come back here if the people there consider it a bug.
+   If you've just upgraded to 9.0, please read the upgrade guide first (https://docs.automapper.org/en/latest/9.0-Upgrade-Guide.html).
+   Try to provide [a minimal, complete, and verifiable example](https://stackoverflow.com/help/mcve), preferably a [gist]   (https://gist.github.com/lbargaoanu/9c7233441c3a3413cc2b9b9ebb5964a9) that we can execute and see fail. [Here](https://gist.github.com/lbargaoanu/0cbc531306223f7ffc5468becf2642d6) is an example for ProjectTo.
```

Comment:
The https://gist.github.com/lbargaoanu/0cbc531306223f7ffc5468becf2642d6 is using the removed `Mapper.Instance`

Reply:
Thanks, I'll change it.